cytominer_eval/operations/mp_value.py

```diff
@@ -0,0 +1,56 @@
+"""
+Functions to calculate multidimensional perturbation values
+
+This describes the distance in dimensionality-reduced space between a perturbation
+and a control.
+"""
+
+import numpy as np
+import pandas as pd
+from typing import List
+
+from .util import calculate_mp_value
+
+
+def mp_value(
+    df: pd.DataFrame,
+    control_perts: List[str],
+    replicate_id: str,
+    features: List[str],
+    params={},
```

Comment:
```suggestion
    params: dict = {},
```

to keep type hints consistent